editor-components/feedback/feedbackmodal1/feedbackmodal1.tsx

```diff
@@ -0,0 +1,212 @@
+import * as React from "react";
+import { BaseModal } from "../../EditorComponent";
+import styles from "./feedbackmodal1.module.scss";
+import { object, string } from "yup";
+import { ComposerIcon } from "../../../composer-base-components/icon/icon";
+
+type Emoji = {
+  type: "icon";
+  key: string;
+  displayer: string;
+  value: string;
+};
```

Comment:
invalid type.
[please read](https://github.com/Teknodev/composer-tools/tree/main/editor-components#how-to-approach-props)